lib/compiler/fmt.zig

```diff
@@ -236,7 +236,7 @@ fn fmtPathDir(
     while (try dir_it.next()) |entry| {
         const is_dir = entry.kind == .directory;
 
-        if (is_dir and (mem.eql(u8, entry.name, "zig-cache") or mem.eql(u8, entry.name, "zig-out"))) continue;
+        if (mem.startsWith(u8, entry.name, ".")) continue;
```

Comment:
Am I wrong in thinking it might still make sense to ignore `zig-out` here?

Reply:
Although zig-out is the default installation prefix, it is not the only one that could possibly exist. It is not supposed to be special.